plugins/native/src/main/java/org/qbicc/plugin/native_/NativeInfo.java

```diff
@@ -189,13 +193,14 @@ ValueType resolveNativeType(final DefinedTypeDefinition definedType) {
                                 if (result != null) {
                                     CProbe.Type.Info typeInfo = result.getTypeInfo(probeType);
                                     long size = typeInfo.getSize();
+                                    int align = annotatedAlign != 0 ? annotatedAlign : (int) typeInfo.getAlign();
```

Comment:
Should we validate that the annotatedAlign is at least as large as `typeInfo.getAlign()`?

Reply:
Not sure if we have to give it much weight, but the comment on the @align annotation was:
```
    /**
     * Explicitly specify the exact alignment of an object.  The alignment will not be probed.  Different alignments can be specified
     * for different platforms.
     */
```
So, I was trying to implement those semantics....if there's an align annotation we just do what it says.  Whether or not that's the right semantics 🤷‍♂️

Reply:
That makes sense.